added 'failing' to options list

BenSower opened the pull request:
See https://github.com/avajs/ava#api

BenSower committed:
added 'failing' to options list

sheerun commented:
It was supposed to be `failing: {failing: true}`.

I've fixed, added tests, and released as 1.1.0

BenSower commented:
Oh nice!

Yeah, stupid copy and paste stuff...
Thanks for the fast merge!

take care!

2017-01-12 18:00 GMT+01:00 Adam Stankiewicz <notifications@github.com>:

> It was supposed to be failing: {failing: true}.
>
> I've fixed, added tests, and released as 1.1.0
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sheerun/ava-spec/pull/8#issuecomment-272219540>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ABGrl5yQLSz0y85wL2oquuygqf0PpSZqks5rRlwRgaJpZM4LYGT0>
> .
>



-- 
[image: Emvolution]

Benjamin Sauer
Emvolution | Geschäftsführer und Head of Technology
E-Mail: ben@emvolution.me
Web: https://b2b.emvolution.me
Blog: Emvolution Story Board <https://blog.emvolution.me/>
